### DISCUSSION TITLE
I am trying to push a helm package to an already created acr on azure sub

### DISCUSSION THREAD

**Discussion by sushmak295** [oras-project/oras-go#158] [General]
please point me to any examples so I can refer for usage of Oras client library. I need a equivalent api for **az acr helm push**

**Comment by FeynmanZhou** [comment]
Hi @sushmak295 , ORAS can push/pull artifacts like Helm charts to any OCI-compatible registry, see [Helm blog](https://helm.sh/blog/storing-charts-in-oci/). 

Currently, ORAS has not been integrated into Azure CLI, you can try it with ORAS CLI directly, see this [guide](https://helm.sh/blog/storing-charts-in-oci/) for reference.

**Comment by dtzar** [comment]
@sushmak295 - what do you feel is missing from [the experience here using just helm](https://docs.microsoft.com/azure/container-registry/container-registry-helm-repos#push-chart-to-registry-as-oci-artifact)?
